products/error_tracking/backend/max_tools.py

```diff
@@ -205,3 +210,69 @@ async def _arun_impl(self, instructions: str) -> tuple[str, ErrorTrackingIssueIm
             except Exception as e:
                 raise ValueError(f"Failed to generate ErrorTrackingIssueImpactToolOutput: {e}")
         return content, events
+
+
+class ErrorTrackingExplainIssueArgs(BaseModel):
+    """TODO: is this needed?"""
```

Comment:
@PostHog/team-posthog-ai I wasn't sure if this was necessary if I had no attributes. Should I be passing args like this or relying on the `context` from the frontend like I am